modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -86,37 +86,43 @@ type: kubernetes.io/dockerconfigjson
 
 [NOTE]
 ====
-* **Automatic Secret Linking via UI**  
-  The {ProductName} UI automatically links image pull secrets to the `appstudio-pipeline` ServiceAccount.  
-  - No pipeline YAML changes are required.  
-  - Tasks like the buildah task will use this secret automatically when pulling images from `quay.io`. (https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.3/buildah-oci-ta.yaml in volumes) 
+* **Automatic Secret Linking via UI**
+  The {ProductName} UI automatically links image pull secrets to the `appstudio-pipeline` ServiceAccount.
+  - No pipeline YAML changes are required.
+  - Tasks like the buildah task will use this secret automatically when pulling images from `quay.io`. (https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.3/buildah-oci-ta.yaml in volumes)
 
-* **Manual Secret Creation**  
-  . **Link Secret To SA** 
+* **Manual Secret Creation**
+  . **Link Secret To SA**
     - If you create the secret manually (e.g., via `kubectl` or YAML), you must **manually link it** to the `appstudio-pipeline` ServiceAccount in your namespace.
-  . **Secrets Via Workspace** 
+  . **Secrets Via Workspace**
     - Explicitly mount secrets as files using a workspace
     - RBAC: Ensure the ServiceAccount has permission to access the secret (via Role/RoleBinding).
 
-  . **Secrets as Environment Variables** 
+  . **Secrets as Environment Variables**
     - Inject secrets into environment variables
     - RBAC: The ServiceAccount must have `get` permission for the secret.
 
-* **Troubleshooting**  
+* **Troubleshooting**
   For issues with secret linking, review the xref:/troubleshooting/index.adoc#check-if-the-secret-is-linked-to-the-service-account[troubleshooting section].
 ====
 
-== Creating source control secrets
+[#gitlab-source-secret]
```

Comment:
What does this do? Is it to create an anchor?

Reply:
It forces the anchor to have a specific name, regardless of title changes.

Reply:
yes